### DISCUSSION TITLE
.webp - images fields

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16912] [Feature improvement]
.webp file format is missing in the list of supported files in image/multi-image fields.

**Comment by rskrzypczak** [comment]
I do not know what you're talking about. webp is supported
https://github.com/YetiForceCompany/YetiForceCRM/blob/e97dea8f2af761011b3ce008dbb692ce2e379778/app/Fields/File.php#L30

**Comment by s2b-git** [comment]
Tested on git dev and own 6.4. Field is empty. Check in Products module please.
![image](https://user-images.githubusercontent.com/84110093/219579060-c74683d4-7b52-445b-9dbc-f3700aa929b2.png)

**Reply by s2b-git** [reply]
6.4 shows:

![image](https://user-images.githubusercontent.com/84110093/219579392-09d86671-73b8-45c5-9790-2e64b6adab8c.png)

**Comment by rskrzypczak** [comment]
1. None of the above messages indicate that the file format is not supported.
2. On demos there is a limit of the size of added files to 1B.
3. Your last screenshot simply informs you that the internal verifier considered the uploaded file to be potentially dangerous and therefore blocked it.
Try adding another webp.

P.S. During my tests, I found no problems with adding or displaying webp files.
